PHEx/src/background.js

```diff
@@ -1,18 +1,19 @@
 const browser = chrome || browser;
-// debug will switch to redirector on localhost instead of prodigyhacking
+
+// If debug is enabled, the extension will use localhost instead of the Prodigy Hacking Organization CDN.
```

Comment:
nothing big of note, but we don't actually have a CDN

Reply:
We can...

Reply:
> nothing big of note, but we don't actually have a CDN

whatever you call it